### PR TITLE
creates fail safes for certain mtl file formatting 

### PR DESCRIPTION
**Pull request by mprafson**
If an mtl file comes in with materials that either (1) no light contribution for ambient, specular, or diffuse or (2) no opacity, the loader will remap the values to 1

### CODE REVIEW

File: src/loaders.js [hunk 1]
```diff
@@ -20,11 +20,19 @@
 
 const noop = function() {};
 
-export const loadObj = (objPath, materials) => new Promise((resolve, reject) => {
+// remaps opacity from 0 to 1
+const opacityRemap = function(mat) {
+  if (mat.opacity == 0) {
```
Comment (jsantell) [inline src/loaders.js]: nit: strict equality with `mat.opacity === 0`

File: src/loaders.js [hunk 2]
```diff
@@ -34,6 +42,8 @@ export const loadMtl = mtlPath => new Promise((resolve, reject) => {
   const loader = new global.THREE.MTLLoader();
 
   loader.setTexturePath(mtlPath.substring(0, mtlPath.lastIndexOf('/') + 1));
+  // remaps ka, kd, & ks values of 0,0,0 -> 1,1,1
+  loader.setMaterialOptions( { ignoreZeroRGBs: true } );
```
Comment (jsantell) [inline src/loaders.js]: style nit: no spaces outside of the curly `setMaterialOptions({ ignoreZeroRGBs: true })`

**Review by jsantell — APPROVED** [review]
Looks great! Excited to get legit support for Blocks in here. Some small stylish nits that I thought the linter had rules for but I was wrong (can add in a follow up rules for this as well).

Also builds are generally done in separate commits (usually whenever cutting a release) to keep the changes in each commit just including the source -- could you remove the dist/* files from this commit?

Other than those changes, looks great, let's merge it in then!

File: src/loaders.js (hunk 1 above)
Comment (mprafson) [reply src/loaders.js]: done

File: src/loaders.js (hunk 2 above)
Comment (mprafson) [reply src/loaders.js]: done

**Pull request merged**